### DISCUSSION TITLE
Allow getting the style for a element with css attributes.

### DISCUSSION THREAD

**Discussion by j-mendez** [programble/scraper#152] [Ideas]
I think adding the ability to get the style for an element with the correlation of the css would be impactful. 

Currently working on this in another project [kayle](https://github.com/a11ywatch/kayle) with the help of [victor-tree](https://github.com/j-mendez/victor-tree). I think if the logic is done right it could move to a feature. 

This would improve the ability to do certain logic during the select process ex: `style.height = 0` `continue` to the next node etc.

**Comment by j-mendez** [comment]
Tagging for visibility @adamreichold  @cfvescovo.

**Reply by cfvescovo** [reply]
Sorry, I totally forgot about this. If you have any suggestions or even some code ready to merge, feel free to open a PR. I will look into this ASAP

**Reply by cfvescovo** [reply]
P.S. A11yWatch rocks! Great project
